### PR TITLE
[Bulk] Validation: Fix issues 6/13

### PR DESCRIPTION
**Pull request by reemsabaw1**
## Category

- [x] Content fix
- [ ] New article

## Related issues

Global effort to fix build validation errors

## What's in this Pull Request?

Fixed missing descriptions and various other issues in 22 articles.

### CODE REVIEW

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[a80a30a](https://github.com/reemsabaw1/sp-dev-docs/commits/a80a30a6152aee80a9c7e33b6da245604da0b67b)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/schema/elements-element-contenttype.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/elements-element-contenttype.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/elements-element-contenttype?branch=pr-en-us-8158) | [Details](#user-content-8084276dcb3383ccb8db9347aa47c877-2ee20178)
[docs/schema/elements-element-contenttypebinding.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/elements-element-contenttypebinding.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/elements-element-contenttypebinding?branch=pr-en-us-8158) | [Details](#user-content-8c989726e71ddb9668678b6cb23d9044-2ee20178)
[docs/schema/elements-element-delegate-control.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/elements-element-delegate-control.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/elements-element-delegate-control?branch=pr-en-us-8158) | [Details](#user-content-207cdd6d87f4c57fbaf83989a67d18e1-2ee20178)
[docs/schema/elements-element-document-converter.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/elements-element-document-converter.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/elements-element-document-converter?branch=pr-en-us-8158) | [Details](#user-content-105ce46a58bf0a08fa4763375e6b2cca-2ee20178)
[docs/schema/elements-element-event.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/elements-element-event.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/elements-element-event?branch=pr-en-us-8158) | [Details](#user-content-8502ca2fa7b25d4bc2f90cfed4555ca4-2ee20178)
[docs/schema/elements-element-featuresitetemplateassociation.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/elements-element-featuresitetemplateassociation.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/elements-element-featuresitetemplateassociation?branch=pr-en-us-8158) | [Details](#user-content-4b2740ec1ed8d7dd8effbc6a6b484470-2ee20178)
[docs/schema/elements-element-field.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/elements-element-field.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/elements-element-field?branch=pr-en-us-8158) | [Details](#user-content-cf4a81136b9c31bc8345c5f75ae3f266-2ee20178)
[docs/schema/elements-element-list-instance.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/elements-element-list-instance.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/elements-element-list-instance?branch=pr-en-us-8158) | [Details](#user-content-0d802fa42154e8c1a98360bd46567368-2ee20178)
[docs/schema/elements-element-list-template.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/elements-element-list-template.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/elements-element-list-template?branch=pr-en-us-8158) | [Details](#user-content-59baeca28d031c4d0d29e1e63b19451f-2ee20178)
[docs/schema/elements-element-module.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/elements-element-module.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/elements-element-module?branch=pr-en-us-8158) | [Details](#user-content-e6d08114a2896e3b28646715ca87d45b-2ee20178)
[docs/schema/elements-element-property-bag.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/elements-element-property-bag.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/elements-element-property-bag?branch=pr-en-us-8158) | [Details](#user-content-233eb0a9e3db1a76d655d1f45c615d01-2ee20178)
[docs/schema/elements-element-workflow.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/elements-element-workflow.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/elements-element-workflow?branch=pr-en-us-8158) | [Details](#user-content-deb57d6829c4a42ab5ab865d257d5ffb-2ee20178)
[docs/schema/elements-sharepoint-add-in-manifest.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/elements-sharepoint-add-in-manifest.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/elements-sharepoint-add-in-manifest?branch=pr-en-us-8158) | [Details](#user-content-86da921f1b766bc084d9f52391145754-2ee20178)
[docs/schema/elements-web-template.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/elements-web-template.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/elements-web-template?branch=pr-en-us-8158) | [Details](#user-content-e18fd05b2391c98944c3fdcedc036fd5-2ee20178)
[docs/schema/elements-workflow-actions.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/elements-workflow-actions.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/elements-workflow-actions?branch=pr-en-us-8158) | [Details](#user-content-02d407842fe42266e8e5613a1a75f062-2ee20178)
[docs/schema/enabledforscoping-element-managedpropertyinfo-complextypesps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/enabledforscoping-element-managedpropertyinfo-complextypesps15xsdsearchset2.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/enabledforscoping-element-managedpropertyinfo-complextypesps15xsdsearchset2?branch=pr-en-us-8158) | [Details](#user-content-cc2a3dfa4a12845285e5148e86e60ba9-2ee20178)
[docs/schema/entities-element-in-contextdefinition-solutionmanifestdefinitions-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/entities-element-in-contextdefinition-solutionmanifestdefinitions-schema.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/entities-element-in-contextdefinition-solutionmanifestdefinitions-schema?branch=pr-en-us-8158) | [Details](#user-content-2877cf8c81a4093201657c53663efcfc-2ee20178)
[docs/schema/entities-element-in-lobsystem-bdcmetadata-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/entities-element-in-lobsystem-bdcmetadata-schema.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/entities-element-in-lobsystem-bdcmetadata-schema?branch=pr-en-us-8158) | [Details](#user-content-f98017183784d73ddb6a6c7d65f6a858-2ee20178)
[docs/schema/entities-element-in-lobsystem-bdcmetadataresource-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/entities-element-in-lobsystem-bdcmetadataresource-schema.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/entities-element-in-lobsystem-bdcmetadataresource-schema?branch=pr-en-us-8158) | [Details](#user-content-fb64b0f8691b573000d718811b5cde26-2ee20178)
[docs/schema/entity-element-in-entities-bdcmetadata-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/entity-element-in-entities-bdcmetadata-schema.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/entity-element-in-entities-bdcmetadata-schema?branch=pr-en-us-8158) | [Details](#user-content-eec2afe375f05c207d2f6f1439278f7a-2ee20178)
[docs/schema/entity-element-in-entities-bdcmetadataresource-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/entity-element-in-entities-bdcmetadataresource-schema.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/entity-element-in-entities-bdcmetadataresource-schema?branch=pr-en-us-8158) | [Details](#user-content-b74c6cbae1777906f621d486ed340ff6-2ee20178)
[docs/schema/entity-element-in-entities-solutionmanifestdefinitions-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/entity-element-in-entities-solutionmanifestdefinitions-schema.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/entity-element-in-entities-solutionmanifestdefinitions-schema?branch=pr-en-us-8158) | [Details](#user-content-b50962f19dd8930d78d6a54863637166-2ee20178)

<a id="8084276dcb3383ccb8db9347aa47c877-2ee20178"></a>
### [docs/schema/elements-element-contenttype.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/elements-element-contenttype.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="8c989726e71ddb9668678b6cb23d9044-2ee20178"></a>
### [docs/schema/elements-element-contenttypebinding.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/elements-element-contenttypebinding.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="207cdd6d87f4c57fbaf83989a67d18e1-2ee20178"></a>
### [docs/schema/elements-element-delegate-control.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/elements-element-delegate-control.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="105ce46a58bf0a08fa4763375e6b2cca-2ee20178"></a>
### [docs/schema/elements-element-document-converter.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/elements-element-document-converter.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="8502ca2fa7b25d4bc2f90cfed4555ca4-2ee20178"></a>
### [docs/schema/elements-element-event.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/elements-element-event.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="4b2740ec1ed8d7dd8effbc6a6b484470-2ee20178"></a>
### [docs/schema/elements-element-featuresitetemplateassociation.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/elements-element-featuresitetemplateassociation.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="cf4a81136b9c31bc8345c5f75ae3f266-2ee20178"></a>
### [docs/schema/elements-element-field.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/elements-element-field.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="0d802fa42154e8c1a98360bd46567368-2ee20178"></a>
### [docs/schema/elements-element-list-instance.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/elements-element-list-instance.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="59baeca28d031c4d0d29e1e63b19451f-2ee20178"></a>
### [docs/schema/elements-element-list-template.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/elements-element-list-template.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="e6d08114a2896e3b28646715ca87d45b-2ee20178"></a>
### [docs/schema/elements-element-module.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/elements-element-module.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="233eb0a9e3db1a76d655d1f45c615d01-2ee20178"></a>
### [docs/schema/elements-element-property-bag.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/elements-element-property-bag.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="deb57d6829c4a42ab5ab865d257d5ffb-2ee20178"></a>
### [docs/schema/elements-element-workflow.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/elements-element-workflow.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="86da921f1b766bc084d9f52391145754-2ee20178"></a>
### [docs/schema/elements-sharepoint-add-in-manifest.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/elements-sharepoint-add-in-manifest.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="e18fd05b2391c98944c3fdcedc036fd5-2ee20178"></a>
### [docs/schema/elements-web-template.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/elements-web-template.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="02d407842fe42266e8e5613a1a75f062-2ee20178"></a>
### [docs/schema/elements-workflow-actions.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/elements-workflow-actions.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="cc2a3dfa4a12845285e5148e86e60ba9-2ee20178"></a>
### [docs/schema/enabledforscoping-element-managedpropertyinfo-complextypesps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/enabledforscoping-element-managedpropertyinfo-complextypesps15xsdsearchset2.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="2877cf8c81a4093201657c53663efcfc-2ee20178"></a>
### [docs/schema/entities-element-in-contextdefinition-solutionmanifestdefinitions-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/entities-element-in-contextdefinition-solutionmanifestdefinitions-schema.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="f98017183784d73ddb6a6c7d65f6a858-2ee20178"></a>
### [docs/schema/entities-element-in-lobsystem-bdcmetadata-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/entities-element-in-lobsystem-bdcmetadata-schema.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="fb64b0f8691b573000d718811b5cde26-2ee20178"></a>
### [docs/schema/entities-element-in-lobsystem-bdcmetadataresource-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/entities-element-in-lobsystem-bdcmetadataresource-schema.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="eec2afe375f05c207d2f6f1439278f7a-2ee20178"></a>
### [docs/schema/entity-element-in-entities-bdcmetadata-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/entity-element-in-entities-bdcmetadata-schema.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="b74c6cbae1777906f621d486ed340ff6-2ee20178"></a>
### [docs/schema/entity-element-in-entities-bdcmetadataresource-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/entity-element-in-entities-bdcmetadataresource-schema.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="b50962f19dd8930d78d6a54863637166-2ee20178"></a>
### [docs/schema/entity-element-in-entities-solutionmanifestdefinitions-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/entity-element-in-entities-solutionmanifestdefinitions-schema.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/8c4da01d-e430-7fe7-0bea-2c4100cce9e8/202206131510168533-8158/BuildReport?accessString=6ab610a223a55701b3f612b54cc123c0e7f0e95accf8b27c988f716fc565e35d).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/schema/elements-element-contenttypebinding.md [hunk 1]
```diff
@@ -58,7 +59,7 @@ The following sections describe attributes, child elements, and parent elements.
 </colgroup>
 <tbody>
 <tr class="odd">
-<td align="left"><p><span sdata="link"><a href="contenttypebinding-element-contenttypebinding.md">ContentTypeBinding Element (ContentTypeBinding)</a></span></p></td>
+<td align="left"><p><span><a href="contenttypebinding-element-contenttypebinding.md">ContentTypeBinding Element (ContentTypeBinding)</a></span></p></td>
```
Comment (RedZephyr13) [inline docs/schema/elements-element-contenttypebinding.md:62]: We need to turn this HTML table into a Markdown table.

File: docs/schema/enabledforscoping-element-managedpropertyinfo-complextypesps15xsdsearchset2.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ description: "Last modified: March 09, 2015"
   
 ## Element information
 
-|||
+| Element Type | Namespace |
```
Comment (RedZephyr13) [inline docs/schema/enabledforscoping-element-managedpropertyinfo-complextypesps15xsdsearchset2.md:19]: Let's change the headers for this table to "Information/Location" or "Name/Value."

File: docs/schema/entities-element-in-contextdefinition-solutionmanifestdefinitions-schema.md [hunk 3]
```diff
@@ -48,7 +49,7 @@ None.
 </thead>
 <tbody>
 <tr class="odd">
-<td align="left"><p><span sdata="link"><a href="entity-element-in-entities-solutionmanifestdefinitions-schema.md">Entity Element in Entities (SolutionManifestDefinitions Schema)</a></span></p></td>
+<td align="left"><p><span><a href="entity-element-in-entities-solutionmanifestdefinitions-schema.md">Entity Element in Entities (SolutionManifestDefinitions Schema)</a></span></p></td>
```
Comment (RedZephyr13) [inline docs/schema/entities-element-in-contextdefinition-solutionmanifestdefinitions-schema.md:52]: We need to turn this HTML table into a Markdown table.

File: docs/schema/entities-element-in-lobsystem-bdcmetadata-schema.md [hunk 4]
```diff
@@ -49,7 +49,7 @@ None.
 </thead>
 <tbody>
 <tr class="odd">
-<td align="left"><p><span sdata="link"><a href="entity-element-in-entities-bdcmetadata-schema.md">Entity Element in Entities (BDCMetadata Schema)</a></span></p></td>
+<td align="left"><p><span><a href="entity-element-in-entities-bdcmetadata-schema.md">Entity Element in Entities (BDCMetadata Schema)</a></span></p></td>
```
Comment (RedZephyr13) [inline docs/schema/entities-element-in-lobsystem-bdcmetadata-schema.md:52]: We need to turn this HTML table into a Markdown table.

File: docs/schema/entities-element-in-lobsystem-bdcmetadata-schema.md [hunk 5]
```diff
@@ -70,7 +70,7 @@ None.
 </thead>
 <tbody>
 <tr class="odd">
-<td align="left"><p><span sdata="link"><a href="lobsystem-element-in-lobsystems-bdcmetadata-schema.md">LobSystem Element in LobSystems (BDCMetadata Schema)</a></span></p></td>
+<td align="left"><p><span><a href="lobsystem-element-in-lobsystems-bdcmetadata-schema.md">LobSystem Element in LobSystems (BDCMetadata Schema)</a></span></p></td>
```
Comment (RedZephyr13) [inline docs/schema/entities-element-in-lobsystem-bdcmetadata-schema.md:73]: We need to turn this HTML table into a Markdown table.

File: docs/schema/entities-element-in-lobsystem-bdcmetadataresource-schema.md [hunk 6]
```diff
@@ -45,7 +45,7 @@ None.
 </thead>
 <tbody>
 <tr class="odd">
-<td align="left"><p><span sdata="link"><a href="entity-element-in-entities-bdcmetadataresource-schema.md">Entity Element in Entities (BDCMetadataResource Schema)</a></span></p></td>
+<td align="left"><p><span><a href="entity-element-in-entities-bdcmetadataresource-schema.md">Entity Element in Entities (BDCMetadataResource Schema)</a></span></p></td>
```
Comment (RedZephyr13) [inline docs/schema/entities-element-in-lobsystem-bdcmetadataresource-schema.md:48]: We need to turn this HTML table into a Markdown table.

File: docs/schema/entities-element-in-lobsystem-bdcmetadataresource-schema.md [hunk 7]
```diff
@@ -63,7 +63,7 @@ None.
 </thead>
 <tbody>
 <tr class="odd">
-<td align="left"><p><span sdata="link"><a href="lobsystem-element-in-lobsystems-bdcmetadataresource-schema.md">LobSystem Element in LobSystems (BDCMetadataResource Schema)</a></span></p></td>
+<td align="left"><p><span><a href="lobsystem-element-in-lobsystems-bdcmetadataresource-schema.md">LobSystem Element in LobSystems (BDCMetadataResource Schema)</a></span></p></td>
```
Comment (RedZephyr13) [inline docs/schema/entities-element-in-lobsystem-bdcmetadataresource-schema.md:66]: We need to turn this HTML table into a Markdown table.

File: docs/schema/entity-element-in-entities-bdcmetadata-schema.md [hunk 8]
```diff
@@ -145,31 +145,31 @@ The following sections describe attributes, child elements, and parent elements.
 </thead>
 <tbody>
 <tr class="odd">
-<td align="left"><p><span sdata="link"><a href="localizeddisplaynames-element-in-metadataobject-bdcmetadata-schema.md">LocalizedDisplayNames Element in MetadataObject (BDCMetadata Schema)</a></span></p></td>
+<td align="left"><p><span><a href="localizeddisplaynames-element-in-metadataobject-bdcmetadata-schema.md">LocalizedDisplayNames Element in MetadataObject (BDCMetadata Schema)</a></span></p></td>
```
Comment (RedZephyr13) [inline docs/schema/entity-element-in-entities-bdcmetadata-schema.md:148]: We need to turn this HTML table into a Markdown table.

File: docs/schema/entity-element-in-entities-bdcmetadataresource-schema.md [hunk 9]
```diff
@@ -45,25 +45,25 @@ None.
 </thead>
 <tbody>
 <tr class="odd">
-<td align="left"><p><span sdata="link"><a href="localizeddisplaynames-element-in-metadataobject-bdcmetadataresource-schema.md">LocalizedDisplayNames Element in MetadataObject (BDCMetadataResource Schema)</a></span></p></td>
+<td align="left"><p><span><a href="localizeddisplaynames-element-in-metadataobject-bdcmetadataresource-schema.md">LocalizedDisplayNames Element in MetadataObject (BDCMetadataResource Schema)</a></span></p></td>
```
Comment (RedZephyr13) [inline docs/schema/entity-element-in-entities-bdcmetadataresource-schema.md:48]: We need to turn this HTML table into a Markdown table.

File: docs/schema/entity-element-in-entities-solutionmanifestdefinitions-schema.md [hunk 10]
```diff
@@ -86,7 +86,7 @@ The following sections describe attributes, child elements, and parent elements.
 </thead>
 <tbody>
 <tr class="odd">
-<td align="left"><p><span sdata="link"><a href="view-element-in-entityinstancedefinition-solutionmanifestdefinitions-schema.md">View Element in EntityInstanceDefinition (SolutionManifestDefinitions Schema)</a></span></p></td>
+<td align="left"><p><span><a href="view-element-in-entityinstancedefinition-solutionmanifestdefinitions-schema.md">View Element in EntityInstanceDefinition (SolutionManifestDefinitions Schema)</a></span></p></td>
```
Comment (RedZephyr13) [inline docs/schema/entity-element-in-entities-solutionmanifestdefinitions-schema.md:89]: We need to turn this HTML table into a Markdown table.

File: docs/schema/entity-element-in-entities-solutionmanifestdefinitions-schema.md [hunk 11]
```diff
@@ -109,7 +109,7 @@ The following sections describe attributes, child elements, and parent elements.
 </thead>
 <tbody>
 <tr class="odd">
-<td align="left"><p><span sdata="link"><a href="entities-element-in-contextdefinition-solutionmanifestdefinitions-schema.md">Entities Element in ContextDefinition (SolutionManifestDefinitions Schema)</a></span></p></td>
+<td align="left"><p><span><a href="entities-element-in-contextdefinition-solutionmanifestdefinitions-schema.md">Entities Element in ContextDefinition (SolutionManifestDefinitions Schema)</a></span></p></td>
```
Comment (RedZephyr13) [inline docs/schema/entity-element-in-entities-solutionmanifestdefinitions-schema.md:112]: We need to turn this HTML table into a Markdown table.

**Review by andrewconnell — APPROVED** [review]

**Pull request merged**